src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/get_whitelisted_indices.js

```diff
@@ -0,0 +1,48 @@
+import { MAX_NUMBER_OF_MATCHING_INDICES } from '../constants';
+
+function whitelist(indices, isIncludingSystemIndices) {
+  if (!indices) {
+    return indices;
+  }
+
+  const acceptableIndices = isIncludingSystemIndices
+    ? indices
+    // All system indices begin with a period.
+    : indices.filter(index => !index.name.startsWith('.'));
+
+  return acceptableIndices.slice(0, MAX_NUMBER_OF_MATCHING_INDICES);
+}
+
+export function getWhitelistedIndices(indices, isIncludingSystemIndices, query, matchingIndices) {
```

Comment:
The word "whitelisted" threw me off in this function name. I kept thinking that it was responsible for whitelisting somehow, but I think that's actually just an implementation detail we can ignore from an interface standpoint. Since you're passing in some initial indices and then getting back different lists of indices which match to various degrees, how about renaming this function to `matchIndices`?

Reply:
Changed!